recipes/object-map.md

```diff
@@ -0,0 +1,73 @@
+import Example from '../components/md/Example';
+
+<Example reactnative>
+
+Defining Object
+```js
+const exampleObject = {
+    type: 'example',
+    value: 3,
+    children: null,
+    isEditable: false
+}
+```
+
+Accessing Object values
+```js
+exampleObject.type;
+exampleObject.value;
+```
+
+</Example>
+
+<Example flutter>
+
+In Dart, there is Map like JS Objects but it is a little bit different
```

Comment:
`Map like JS Objects` is a bit confusing statement. JS objects are actual maps (if we're not speaking about shadow classes which are created under the hood), so I'd drop this comment completely